scripts/buildLogos.js

```diff
@@ -40,6 +44,12 @@ async function buildLogos() {
 
   // eslint-disable-next-line no-restricted-globals,compat/compat,promise/no-native
   await Promise.all(logoPromises);
+
+  if (shouldCommit) {
+    await $`git add .`;
```

Comment:
```suggestion
    await $`git add cdn`;
```

Probably safer to add the cdn folder specifically to avoid pushing other changes unintentionally